app/views/foreman_ansible/job_templates/capsule_upgrade_-_ansible_default.erb

```diff
@@ -47,7 +45,7 @@ feature: ansible_run_capsule_upgrade
                                "--whitelist=#{input('whitelist_options')}"
                              end -%>
       - name: Upgrade Capsule server using satellite-maintain
-        shell: satellite-maintain upgrade run --assumeyes --target-version={{ target_version }} <%= whitelist_option %>
+        shell: satellite-maintain upgrade run --assumeyes <%= whitelist_option %>
```

Comment:
What does it do now? Does it go up a version or all the way to latest available?